CesiumUtility/include/CesiumUtility/SharedAsset.h

```diff
@@ -103,20 +103,23 @@ class CESIUMUTILITY_API SharedAsset : public CesiumUtility::ExtensibleObject {
    * useful to assign the data in derived classes.
    */
   SharedAsset& operator=(const SharedAsset& rhs) {
-    CesiumUtility::ExtensibleObject::operator=(rhs);
+    if (rhs != this) {
```

Comment:
This doesn't look right, comparing an instance to a pointer. Shouldn't it be:
```suggestion
    if (&rhs != this) {
```

Same with the move assignment operator below.